script/example.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+import yaml
+import json
+from argparse import ArgumentParser
+
+def action_json(args):
+
+    with open('scripts/example_data.json') as json_file:
```

Comment:
It's more efficient to use `json.load()` directly instead of `json.loads()` with `file.read()` to avoid reading the entire file into memory.
```suggestion
with open('scripts/example_data.json') as json_file:
    print(json.dumps(json.load(json_file), indent=2))
```